scripts/docker_deploy_manual.sh

```diff
@@ -19,13 +19,13 @@ cd "$DIR"
 git clone --depth 2 https://github.com/ethereum/solidity.git -b "$branch"
 cd solidity
 commithash=$(git rev-parse --short=8 HEAD)
-echo -n "$commithash" > commit_hash.txt
+# echo -n "$commithash" > commit_hash.txt
 version=$("$(dirname "$0")/get_version.sh")
 if [ "$branch" = "release" ] || [ "$branch" = v"$version" ]
 then
-    echo -n > prerelease.txt
+    rm cmake/prerelease.lock
 else
-    date -u +"nightly.%Y.%-m.%-d" > prerelease.txt
+    date -u +"nightly.%Y.%-m.%-d" > cmake/prerelease.lock
 fi
```

Comment:
Putting date in `prerelease.txt` won't work. You should replace this with cmake flags.